### ISSUE TITLE
iceConnectionState changed to disconnected on the connection 

### ISSUE THREAD

**Issue by Rajmanic1999** [peers/peerjs#883]
HI,
  I face issues for PeerJs call established but  iceConnectionState changed
PeerJS:  iceConnectionState changed to disconnected on the connection with 8160e2bc-2234-4012-XXXX XXXX

**Comment by Rajmanic1999** [comment]
i found one solution, 

STUN + TURN  add config webRTC

https://gist.github.com/yetithefoot/7592580
https://stackoverflow.com/questions/34405816/peerjs-webrtc-iceconnectionstate-failed

**Comment by szcuipeng** [comment]
I got the same log in chrome console.

I have two html pages:
server.html
client.html

The client successfully  connected to server using peer.connect ( data channel ),
I got a log  "iceConnectionState changed to disconnected on the connection with xxxx" when i closed server.html.

but there were not "close" or "error" event on client connection, is this a bug or a feature? i am not very familer with WebRTC data channel. i wanna to add reconnect and heartbeat to peerjs's data channel. how to?

any advice will be appreciated!
